vidify/player/web.py

```diff
@@ -0,0 +1,83 @@
+"""
+"""
+
+import logging
+from typing import Optional
+from threading import Thread
+from webbrowser import open
+from threading import Timer, Thread
+
+try:
+    import flask
+except OSError:
+    raise ImportError("Flask is not installed")
+
+from vidify import Platform, CURRENT_PLATFORM
+from vidify.player.generic import PlayerBase
+from PyQt5.QtCore import QObject
+import sys
+import os
+
+PORT = 9999
```

Comment:
We'll have to add a config option for the port later for whenever this one is already being used.